lib/csrf.js

```diff
@@ -86,20 +98,21 @@ module.exports = function (options) {
         }
     }
 
-
     return function checkCsrf(req, res, next) {
 
         var shouldBypass = false;
 
         if (blocklist) {
-            blocklist.some(function (exclusion) {
+            shouldBypass = blocklistExact.has(req.path) ||
+            Array.from(blocklistIncludes).some(function (exclusion) {
```

Comment:
I think instead of creating array in each request, you can either build Array while initializing, or use `for..of` iterator on the Set.

Reply:
👍 Updated to create array during [initialization](https://github.com/krakenjs/lusca/pull/145/files#diff-8c9536c7d8cf45aaa6b95b8ebc315c7e173505c4ce6969789d8ec77a258e66aaR60). Decided not to build array directly because:
- Converting to Set would remove any possible duplicates in the config
- Code seems to be more cleaner/more readable